texar/modules/connectors/connectors_test.py

```diff
@@ -0,0 +1,200 @@
+#
+"""
+Unit tests for connectors.
```

Comment:
Could you add test cases where the output size is a tuple of `int`, and cases where it is a tuple with mixed `torch.Size` and `int` (not sure if we intend to support this)?

Reply:
Yeah actually I don't think the code should support mixture of types as output size, if user wants to use `torch.Size` and `int`, they may wrap `int` into `torch.Size`.